### PR TITLE
Fix crashers from 6.0.4 and improve long-tap recognition

### PR DESCRIPTION
**Pull request by sreilly**

### CODE REVIEW

**Comment by nilnilnil** [comment]
@sreilly who is a good reviewer for these changes? Typically we always have at least a spot checker. In this case, it's a hotfix, so this is cool; but a retroactive review would be nice.

**Comment by sreilly** [comment]
@nilnilnil  Hmm... great question.  If it's ok with you I can ask @mark5422 to take a look at future PRs.  If you've got someone on your side who you'd prefer that would be fine too.  I do like the idea of a spot checker for all of these sorts of updates.

**Comment by nilnilnil** [comment]
@sreilly I'm fine either way. I do think we should employ reviews more.

File: DuckDuckGo/DDGWebViewController.m [hunk 1]
```diff
@@ -148,10 +149,10 @@ - (void)viewDidLoad
     UIMenuItem *search = [[UIMenuItem alloc] initWithTitle:NSLocalizedString(@"Search", @"Search") action:@selector(search:)];
     [[UIMenuController sharedMenuController] setMenuItems:@[search]];
     
-    UITapGestureRecognizer *tapGestureRecognizer = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(handleTap:)];
-    tapGestureRecognizer.delegate = self;
-    tapGestureRecognizer.numberOfTapsRequired = 1;
-    [self.webView addGestureRecognizer:tapGestureRecognizer];
+    self.tapGestureRecognizer = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(handleTap:)];
+    self.tapGestureRecognizer.delegate = self;
+    self.tapGestureRecognizer.numberOfTapsRequired = 1;
```
Comment (mark5422) [inline DuckDuckGo/DDGWebViewController.m:154]: Setting the numberOfTapsRequired is not necessary as the UITapGestureRecognizer's default for this is 1.

**Pull request merged**